Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Can you check your editor config, this looks to have changed the file encoding (UTF8 w/BOM to w/o BOM or the other way around)

Reply:
So it seems like it automatically removes the BOM if it's not needed.